druid/src/main/java/org/apache/calcite/adapter/druid/DruidExpressions.java

```diff
@@ -159,7 +159,7 @@ public static String toDruidExpression(
   }
 
   public static String fromColumn(String columnName) {
-    return DruidQuery.format("\"%s\"", columnName);
+    return DruidQuery.format("%s", columnName);
```

Comment:
I did't run a test but I guess we may lose unicode identifier support here. You can see how Druid handles unicode identifier from Druid expression's grammer code[1] and the tree listener code[2].

[1] https://github.com/apache/incubator-druid/blob/1f299408114d35a7101eb59fdcc3b6c9e30d8286/core/src/main/antlr4/org/apache/druid/math/expr/antlr/Expr.g4#L36
[2] https://github.com/apache/incubator-druid/blob/1f299408114d35a7101eb59fdcc3b6c9e30d8286/core/src/main/java/org/apache/druid/math/expr/ExprListenerImpl.java#L315

Reply:
I'm not exactly sure what you sent me. The problem I'm trying to fix is
`{
      "type": "expression",
      "name": "vc",
      "expression": "timestamp_parse(timestamp_format(\"__time\",'yyyy-MM-dd\\u0027T\\u0027HH:mm:ss.SSS\\u0027Z\\u0027','America/New_York'),'','UTC')",
      "outputType": "LONG"
    }`
Where the column __time is is incorrectly formatted as \"__time\". I've issued the json myself with the escaped double quotes and it does not work, which is what prompted the change.

The Druid docs say 

> Expressions can contain variables. Variable names may contain letters, digits, '_' and '$'. Variable names must not begin with a digit. **To escape other special characters, you can quote it with double quotation marks.**

The expression call is already inside a json string. So I'm not sure it needs escape characters

Reply:
> The expression call is already inside a json string. So I'm not sure it needs escape characters

Sorry I am a little confused with this. What I'm seeing is that your change removes the double quote of identifiers from the generated Druid expression. Suppose previously the generated expression is 
```
timestamp_format("\\u0000\\u0001\\u0002",'yyyy-MM-dd','America/New_York')
```
, now it would become
```
timestamp_format(\\u0000\\u0001\\u0002,'yyyy-MM-dd','America/New_York')
```
Obviously it is an invalid expression because the parser[1] does not accept a `\\u0000\\u0001\\u0002` as identifier.

[1] https://github.com/apache/incubator-druid/blob/1f299408114d35a7101eb59fdcc3b6c9e30d8286/core/src/main/antlr4/org/apache/druid/math/expr/antlr/Expr.g4#L36

Reply:
Also I don't really understand why `__time` is different with `"__time"`, it seems that they should be equal. I'll run a test as soon as possible.

Reply:
The issue is that it isn't `__time` vs `"__time"` it's `__time`  vs `\"__time\"` it is literally writing the backslashes into the expression call. Remember that the expression itself is written inside quotes

`{"expression":"timestamp_format(__time, 'yyyy-MM-dd', 'America/New_York')"}`

Per the [Druid documentation](http://druid.io/docs/latest/misc/math-expr.html) Single quotes are for String literals, which is what is used for the date format and timezone. It says to use double quotes to escape special characters, which I think this is what we are talking about, but I don't know if that applies to column names. The issue I found with the escaped double quotes was that it was being applied to the column name and putting in the backslash into the json string. So of course Druid couldn't find a column named `\"__time\"` it doesn't exist